### ISSUE TITLE
Detect extra space before expression modifier keywords

### ISSUE THREAD

**Issue by bbatsov** [bbatsov/rubocop#2008]
Basically code like this should be flagged:

```ruby
# two spaces before the if
puts 'hello'  if something
```

@jonas054 Will you take care of this?

**Comment by jonas054** [comment]
Yes, I hope to have some time to spare in a few days.
